spaceship/spec/portal/portal_stubbing.rb

```diff
@@ -254,6 +254,20 @@ def adp_stub_app_groups
         to_return(status: 200, body: adp_read_fixture_file('deleteApplicationGroup.action.json'), headers: { 'Content-Type' => 'application/json' })
     end
 
+    def adp_stub_merchants
```

Comment:
I would love to move away from this style of request stubbing. For an example of the new style, please check out `key_spec.rb` and `portal_client_spec.rb` in https://github.com/fastlane/fastlane/pull/9560/files#diff-427c42b9c7ea4456fc2eee77cf6e4b1b 

The high-level goal with this change is to make the fixtures more flexible by putting them in code rather then brittle fixture files. Also, we now separate the tests into two different concerns. First is making the expected HTTP request ( as in `portal_client_spec.rb`) and the other ensures that the data model (`key_spec.rb`) behaves as expected based on received data from the API.